crates/wash-lib/Cargo.toml

```diff
@@ -82,9 +82,9 @@ wadm = { workspace = true, optional = true }
 walkdir = { workspace = true }
 wascap = { workspace = true }
 wasm-encoder = { workspace = true }
-wasmbus-rpc = { workspace = true }
 wasmcloud-component-adapters = { version = "0.2.4" } # override this version, since 0.3.0 uses an adapter based on wasmtime v0.14
 wasmcloud-control-interface = { workspace = true }
+wasmcloud-core = { workspace = true }
```

Comment:
@rvolosatovs This change means we can't publish wash-lib anymore, until `wasmcloud-core` is published: https://github.com/wasmCloud/wasmCloud/issues/641 is done

Reply:
yes, that crate will be added to the CI setup